test/test_01_02.py

```diff
@@ -0,0 +1,20 @@
+from typing import Sequence
+
+import pytest
+
+from hm_1.tasks.task02 import check_fibonacci
+
+
+@pytest.mark.parametrize(
+    ("values", "expected_result"),
+    [
+        ([1, 1, 2, 3, 5, 8], True),
+        ([1, 1, 3, 4], False),
+        ([1, 1], True),
+        ([1, 1, 2, 3, 5, 8, 13], True),
+    ],
```

Comment:
Please add more cases

- empty list
- single item in the list (fib ad not fib)
- two items list not fib